js/catalog.js

```diff
@@ -12,9 +12,11 @@ function populateForm() {
   //TODO: Add an <option> tag inside the form's select for each product
   const selectElement = document.getElementById('items');
   for (let i in Product.allProducts) {
-
+    let option = document.createElement('option');
+    option.value = Product.allProducts[i].name;
+    option.textContent = Product.allProducts[i].name;
+    option.appendChild(selectElement);
```

Comment:
`selectElement.appendChild(option)` Otherwise you're appending selectElement, which is already in your html, to option, which will not be in your html.